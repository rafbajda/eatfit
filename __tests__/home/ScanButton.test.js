import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import ScanButtonElement from "../../app/screens/home/components/ScanButton";

describe('ScanButton snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<ScanButtonElement />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
