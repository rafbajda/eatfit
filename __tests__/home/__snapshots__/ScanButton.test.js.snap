// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScanButton snapshot renders component 1`] = `
<View
  className="sc-bxivhb sc-VigVT gggsb"
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#007aff",
        "borderBottomWidth": null,
        "borderColor": "#007aff",
        "borderLeftWidth": null,
        "borderRadius": 5,
        "borderRightWidth": null,
        "borderTopWidth": null,
        "borderWidth": undefined,
        "elevation": 2,
        "flexDirection": "row",
        "height": 45,
        "justifyContent": "space-between",
        "opacity": 1,
        "paddingBottom": 6,
        "paddingTop": 6,
        "shadowColor": undefined,
        "shadowOffset": undefined,
        "shadowOpacity": undefined,
        "shadowRadius": undefined,
      }
    }
  >
    <Text
      className="sc-fjdhpX jEoUFb"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#fff",
          "fontFamily": "System",
          "fontSize": 16.5,
          "marginLeft": 0,
          "marginRight": 0,
          "paddingLeft": 16,
          "paddingRight": 16,
        }
      }
      uppercase={false}
    >
      [missing "en.buttons.scan" translation]
    </Text>
  </View>
</View>
`;
